### PR TITLE
Read and write placement information for lyrics

### PR DESCRIPTION
**Pull request by miiizen**
Writes a placement tag for lyrics if the placement is above, and reads lyric placement.
Screenshot 2024-02-22 at 14 36 58

### CODE REVIEW

**Review by oktophonie — APPROVED** [review]

**Comment by Jojo-Schmitz** [comment]
I'm confused, the ...ref.mscx doesn't have any lyrics placed above the staff, it just has stem directions up (for voice 1) and down (for voice 2), for no good reason (as that's the default anyhow)?!?

**Comment by Jojo-Schmitz** [comment]
The import of the xml file looks correct (in the CI artifact), but saving and reopening does not, all lyrics are below the staff, so no wonder the test succeeds...

**Comment by Jojo-Schmitz** [comment]
> Writes a placement tag for lyrics if the placement is below, and reads lyric placement.

No, it writes a placement tag for lyrics if the placement is **above**, and reads lyric placement.

File: src/importexport/musicxml/tests/data/testLyricPos_ref.mscx [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+            <Note>
+              <pitch>55</pitch>
+              <tpc>15</tpc>
+              </Note>
+            </Chord>
+          <Chord>
+            <durationType>quarter</durationType>
+            <Lyrics>
+              <text>above</text>
+              </Lyrics>
```
Comment (Jojo-Schmitz) [inline src/importexport/musicxml/tests/data/testLyricPos_ref.mscx:153]: Should be:
```xml
          <Chord>
            <durationType>quarter</durationType>
            <Lyrics>
              <placement>above</placement>
              <text>Lyrics</text>
              </Lyrics>
            <StemDirection>up</StemDirection>
            <Note>
              <pitch>55</pitch>
              <tpc>15</tpc>
              </Note>
            </Chord>
```

File: src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp [hunk 2]
```diff
@@ -6164,6 +6165,9 @@ void MusicXMLParserLyric::parse()
         lyric->setPropertyFlags(Pid::COLOR, PropertyFlags::UNSTYLED);
     }
     lyric->setVisible(printLyric);
+    if (!placement.empty()) {
+        lyric->setPlacement((placement == "above" ? PlacementV::ABOVE : PlacementV::BELOW));
```
Comment (Jojo-Schmitz) [inline src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp:6169]: double ((...)) ?

Comment (Jojo-Schmitz) [reply src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp:6169]: I believe there's 
```cpp
            lyric->setPropertyFlags(Pid::PLACEMENT, PropertyFlags::UNSTYLED);
```
missing

**Review by Jojo-Schmitz — APPROVED** [review]
Looks good now

**Review by mike-spa — APPROVED** [review]

**Pull request merged**